assignments/this.js

```diff
@@ -1,26 +1,83 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit Binding
+* 2. Explicit Binding
+* 3. new Binding
+* 4. window Binding
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
-
+//Window Binding: can use window.xyz to change the value you see
 // code example for Window Binding
-
-// Principle 2
-
-// code example for Implicit Binding
-
-// Principle 3
-
-// code example for New Binding
-
-// Principle 4
-
-// code example for Explicit Binding
\ No newline at end of file
+var sayRenter = function(){
+    console.log(this.lastRenter);
+  };
+  
+  var scoot315 = {
```

Comment:
Please use let or const, in this case because there should only ever be one `scoot315 =` statement and it is of type object. Const would be most appropriate.

Reply:
scoot315 doesn't fit right here for this example